nextvibe/configs/settings/development.py

```diff
@@ -0,0 +1,19 @@
+from .base import *
+import environ
+
+environ.Env.read_env(os.path.join(BASE_DIR, 'env', '.env.dev'))
```

Comment:
❗ env_example 폴더에 `.env.development` 파일을 만들었는데, 여기서는 `.env.dev` 파일을 참조하고 있어요. 통일해 주세요.
```suggestion
environ.Env.read_env(os.path.join(BASE_DIR, 'env', '.env.development'))
```